src/system/ExtensionsModule/Form/Type/ExtensionModifyType.php

```diff
@@ -73,6 +74,10 @@ public function getBlockPrefix()
 
     public function configureOptions(OptionsResolver $resolver)
     {
+        $resolver->setRequired('extensionType');
+        $resolver->setAllowedTypes('extensionType', 'string');
+        $resolver->setAllowedValues('extensionType', ['module', 'theme']);
+
         $resolver->setDefaults([
             'data_class' => ExtensionEntity::class
```

Comment:
what about adding default = module?

Reply:
changed in 8497b909e99126073611575b84a3147c0ac78551

Reply:
interesting. I thought it belonged in the `setDefaults` method. looks fine.

Reply:
oh yes, this is even better; will move it

Reply:
changed in 9f0f0de281d68b18dc178c76f751559f77678e45